docker/Dockerfile

```diff
@@ -32,3 +32,5 @@ RUN pip3 --no-cache-dir install --upgrade pip \
     && pip3 --no-cache-dir install flashinfer -i https://flashinfer.ai/whl/cu121/torch2.4/
 
 ENV DEBIAN_FRONTEND=interactive
+
+ENTRYPOINT ["python3", "-m", "sglang.launch_server"]
```

Comment:
Is this a good practice? @vhain @zhyncs 
I want to avoid this because this hides some details and we way change the API later.

Reply:
> I want to avoid this because this hides some details and we way change the API later.

agree

Reply:
Make sense, I'll revert it.